tests/test_text.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+# Copyright FMR LLC <opensource@fidelity.com>
+# SPDX-License-Identifier: GNU GPLv3
+import pandas as pd
+
+from feature.selector import Selective, SelectionMethod
+from test_base import BaseTest
+from textwiser import TextWiser, Embedding, PoolOptions, Transformation, WordOptions
+
+
+
```

Comment:
Per PEP-8 standard, we use 2 blank lines between Class definitions and 1 blank line between methods. If you are using PyCharm it should be underlining these python standard errors